resources/views/dashboard/controllers/ticket.blade.php

```diff
@@ -21,6 +21,8 @@
             <p><b>Start Time:</b> {{ $ticket->start_time }}</p>
             <p><b>End Time:</b> {{ $ticket->end_time }}</p>
             <p><b>Duration:</b> {{ $ticket->duration }}</p>
+	        <p><b>Score (1-5):</b> @if($ticket->score) {{ $ticket->score }} @else N/A @endif</p>
```

Comment:
For UX, we need to provide some context behind the score (ex. 5 is best, 1 is worst)